visual-java/src/main/java/com/saucelabs/visual/VisualApi.java

```diff
@@ -160,7 +178,7 @@ public VisualApi(
           "Invalid SauceLabs credentials. "
               + "Please check your SauceLabs username and access key at https://app.saucelabs.com/user-settings");
     }
-    this.client = new GraphQLClient(url, username, accessKey);
+    this.client = new GraphQLClient(resolveEndpoint(url), username, accessKey);
```

Comment:
As mentioned in my other comment, I'm not super convinced by invoking `resolveEndpoint` here.

IMO `resolveEndpoint` should only be invoked when there is not value available (I mean when the function invoked does not have this param).

Any opinion @konraddysput ?